include/base_interface.h

```diff
@@ -189,6 +189,7 @@ class BaseInterface : public ParameterAcceptor
    */
   virtual void compute_system_operators(const std::vector<shared_ptr<typename LATrilinos::BlockMatrix> >,
```

Comment:
and add an explanation/example in the base interface. Say with jacobi and SOR. Including an explanation of what parameter should be set to use the second preconditioner.